src/coverdb.rs

```diff
@@ -29,7 +31,7 @@ table! {
         Format -> Integer,
         Width -> Integer,
         Height -> Integer,
-        Data -> Nullable<Binary>,
+        Data -> Integer,
```

Comment:
Urgh! Dont do that!
Just insert None.

Reply:
Unfortunately that results in a different entry in the sqlite db. (NULL vs 0)